src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ReportDao;
+import core.basesyntax.dao.ReportDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.BalanceSetter;
+import core.basesyntax.service.ConverterFruitTransaction;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.ReportCreate;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.TransactionProcess;
+import core.basesyntax.service.impl.BalanceSetterImpl;
+import core.basesyntax.service.impl.ConverterFruitTransactionImpl;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.ReportCreateImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.TransactionProcessImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseOperation;
+import core.basesyntax.strategy.ReturnOperation;
+import core.basesyntax.strategy.SupplyOperation;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String TRANSACTIONS_FILE_PATH
+            = "src/main/resources/read.csv";
+    private static final String REPORT_FILE_PATH
+            = "src/main/resources/report.csv";
+    private static final Map<Operation, OperationHandler> operations =
+            Map.of(Operation.SUPPLY, new SupplyOperation(),
+                    Operation.PURCHASE, new PurchaseOperation(),
+                    Operation.RETURN, new ReturnOperation());
```

Comment:
Let's move map initialization to the `main` method
```suggestion
    private static final Map<Operation, OperationHandler> operations =
            Map.of(Operation.SUPPLY, new SupplyOperation(),
                    Operation.PURCHASE, new PurchaseOperation(),
                    Operation.RETURN, new ReturnOperation(),
                    Operation.BALANCE, new BalanceOperation());
```